examples/soap-calculator/test/acceptance/application.acceptance.ts

```diff
@@ -0,0 +1,80 @@
+import {supertest} from '@loopback/testlab';
+import {SoapCalculatorApplication} from '../../src/application';
+import {expect} from '@loopback/testlab';
+import {givenAConnectedDataSource} from '../helpers';
+
+describe('Application', function() {
+  let app: SoapCalculatorApplication;
+  let client: supertest.SuperTest<supertest.Test>;
+
+  // tslint:disable-next-line:no-invalid-this
+  this.timeout(10000);
+
+  before(givenAnApplication);
+  before(givenAConnectedDataSource);
```

Comment:
I don't _think_ we need to set up a connected datasource here anymore.

Reply:
Thanks @shimks  , I will update  it.